### PR TITLE
Limit # of imports at the same time per user

### LINKED ISSUE #2132: [limits] # of imports at the same time
we should limit the number of imports per person at the same time (and the ones in the queue)

### PR DESCRIPTION
**Pull request by Kartones**
Fixes #2132

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1630/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1633/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1639/))

**Comment by Kartones** [comment]
retest this please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1641/))

**Comment by Kartones** [comment]
@rafatower CR of the backend please

**Comment by javierarce** [comment]
@CartoDB/frontend: can you review the js part, please?

I modified the code to show the `429` error message that comes from the backend: 

* https://github.com/CartoDB/cartodb/commit/d569ed565f31f25beddb1a2a0bfcfed95e6d8c0b
* https://github.com/CartoDB/cartodb/commit/ebce67c4a291a3d48e5acdb6fb4caf62beb56f96

**Comment by javierarce** [comment]
And here's the result, for @CartoDB/design:

screen shot 2015-02-17 at 10 02 29
screen shot 2015-02-17 at 10 35 39

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1644/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1645/))

File: services/platform-limits/lib/importer/table_row_count.rb [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+        attr_accessor :db
+
+        def subkey
+          'Importer:TableRowCount'
+        end
+
+        # @param context mixed
+        # @return bool
+        def is_over_limit(context)
```
Comment (rafatower) [inline services/platform-limits/lib/importer/table_row_count.rb]: document what the context has for concrete classes at the top of the file

**Comment by rafatower** [comment]
backend looks good :+1:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1687/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1694/))

**Comment by Kartones** [comment]
@CartoDB/frontend PING again, sorry to insist but I need to push this live as soon as possible and is only pending the FE CR

**Comment by xavijam** [comment]
Checking

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1724/))

**Pull request merged**